### PR TITLE
add-character

### PR DESCRIPTION
**Pull request by RasmusAlbrektsen**
Added the methods and attributes to characters, read in the diagram.
Error in the arraylist, since the "Item"-class hasn't been created yet.

### CODE REVIEW

File: src/worldofzuul/Character.java [hunk 1]
```diff
@@ -10,6 +10,12 @@
  * @author Kasper
  */
 public class Character {
+    
+    
+    private Room currentRoom;
```
Comment (quartercastle) [inline src/worldofzuul/Character.java]: Missing comment.

File: src/worldofzuul/Character.java [hunk 2]
```diff
@@ -29,4 +35,30 @@ public Character(String name){
```
Comment (quartercastle) [inline src/worldofzuul/Character.java]: There is no need for a no args constructor.

File: src/worldofzuul/Cook.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+   private String name = "Cook";
+
+   /**
+    * The method gets a steak.
+    */
+   public void getSteak() {
+   
+       
+       
+}
```
Comment (quartercastle) [inline src/worldofzuul/Cook.java]: Need to be indented once.

Comment (quartercastle) [reply src/worldofzuul/Cook.java]: Maybe just remove the method until we need it.

File: src/worldofzuul/Cook.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+/**
+ *
+ * @author Rasmus
+ */
+public class Cook extends Character {
+    
+    /**
+     * Names the character "Cook".
+     */
+   private String name = "Cook";
```
Comment (quartercastle) [inline src/worldofzuul/Cook.java]: Think a better solution would be to set the name with a constructor.
```java
public Cook () {
     super("Cook");
}
```

File: src/worldofzuul/Player.java [hunk 5]
```diff
@@ -5,11 +5,47 @@
...
+import java.util.ArrayList;
+
 /**
  *
  * @author Kasper
  */
 public class Player extends Character{
+ /**
+  * The arraylist stores the items, that the player picks up. 
+  */
```
Comment (quartercastle) [inline src/worldofzuul/Player.java]: The docblock should be indented to the method.

File: src/worldofzuul/Player.java [hunk 6]
```diff
@@ -5,11 +5,47 @@
...
+    /**
+     * The method adds items to the arraylist.
+     * @param items the item which is added to the inventory.
+     */
+    public void addItems(ArrayList items){
+        
+    }
+     /**
+      * The method displays the items currently held.
+      */
```
Comment (quartercastle) [inline src/worldofzuul/Player.java]: indentation should be fixed.

File: src/worldofzuul/Trump.java [hunk 7]
```diff
@@ -14,5 +14,12 @@
      * The name of the character
      */
     private String name = "Donald Trump";
```
Comment (quartercastle) [inline src/worldofzuul/Trump.java]: Think a better solution would be to set the name with a constructor.
```java
public Trump () {
     super("Donald Trump");
}
```

File: src/worldofzuul/Character.java (hunk 2 above)
Comment (RasmusAlbrektsen) [reply src/worldofzuul/Character.java]: Fixed in e5d88f3b36c3505ecf61be143de7681bea18ff7e

Comment (RasmusAlbrektsen) [reply src/worldofzuul/Character.java]: Fixed in 989838424f15261c1ef4628aef6188d1cf2fcd5d

File: src/worldofzuul/Cook.java (hunk 4 above)
Comment (RasmusAlbrektsen) [reply src/worldofzuul/Cook.java]: Fixed in e5d88f3b36c3505ecf61be143de7681bea18ff7e

File: src/worldofzuul/Player.java [hunk 8]
```diff
@@ -5,11 +5,47 @@
...
+    private boolean isInventoryFull() {
+        
+        if (items.length() > 1) {
+            
+            return true;
+        }
+        else {
+            return false;
+        }
+    }
```
Comment (quartercastle) [inline src/worldofzuul/Player.java]: When you have a method with a simple if statement and it just returns a boolean, the if evaluation statement could just be returned instead.
```java
private boolean isInventoryFull() {
     return (items.length() > 1);
}
```

Comment (RasmusAlbrektsen) [reply src/worldofzuul/Player.java]: Fixed in 66ca6fefda976000a256c536838a2fca524eaf2d

**Review by quartercastle — APPROVED** [review]
👍

**Pull request merged**